CHANGELOG.md

```diff
@@ -13,6 +13,34 @@
   
 ### Features
 
+- Screenshots are now supported for Flutter Web via HTML Renderer ([#2361](https://github.com/getsentry/sentry-dart/pull/2361))
```

Comment:
I don't see screenshots when testing with the HTML renderer (`flutter run -d chrome --web-renderer html`):
https://sentry-sdks.sentry.io/issues/6051629474/events/b89bcf0150444b7586205ce8c0afdb42/

Reply:
very strange. you are right. It is not working. Looks like I made a mistake during testing. I probably forgot to change to the `latest` issue in the sentry app. I will revert these changes, later or tomorrow.